cmd/if-mover/main.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
+)
+
+const (
+	ifBase     = "mlink"
+	hostNsFile = "/tmp/hostns.txt"
+	multusLink = "net1"
+)
+
+func main() {
```

Comment:
The daemonset is just using the rook image, right? In that case I would just suggest we add a new command to the existing `rook` binary. I don't see a need for a new binary. 
In this case, the new command would have an entrypoint under `cmd/rook/ceph`, and most of the implementation code would be added perhaps under `pkg/daemon/ceph/multus`